src/Components/CreateImageWizardV2/steps/Packages/Packages.tsx

```diff
@@ -664,6 +848,35 @@ const Packages = () => {
     }
   };
 
+  const handleGroupSelect = (
+    grp: GroupWithRepositoryInfo,
+    _: number,
+    isSelecting: boolean
+  ) => {
+    if (isSelecting) {
+      if (
+        isSuccessEpelRepo &&
+        epelRepo.data &&
+        grp.repository === 'recommended' &&
+        !recommendedRepositories.some((repo) => repo.name?.startsWith('EPEL'))
+      ) {
+        setIsRepoModalOpen(true);
+        setIsSelectingGroup(grp);
+      } else {
+        dispatch(addGroup(grp));
+      }
+    } else {
+      dispatch(removeGroup(grp.name));
+      if (
+        isSuccessEpelRepo &&
+        epelRepo.data &&
+        groups.filter((grp) => grp.repository === 'recommended').length === 1
```

Comment:
When removing last package group from EPEL, it removes the EPEL repo even when there are still packages added from it.

![repo-removed](https://github.com/osbuild/image-builder-frontend/assets/49452678/f689fed9-ef96-4855-82bb-43c29cfbc20c)